WordPress/Tests/Theme/NoDeregisterCoreScriptUnitTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * A sniff unit test checks a .inc file for expected violations of a single
+ * coding standard. Expected errors and warnings are stored in this class.
+ *
+ * @category  Theme
+ * @package   WordPress_Coding_Standards
+ * @author    Simon Prosser <pross@pross.org.uk>
+ */
+class WordPress_Tests_Theme_NoDeregisterCoreScriptUnitTest extends AbstractSniffUnitTest
+{
```

Comment:
Please bring the opening bracket on the same line as the class name as with the other class.